### DISCUSSION TITLE
lichess.org

### DISCUSSION THREAD

**Discussion by MAHcodes** [catppuccin/catppuccin#1831] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Add any other context or screenshots about the port request here.
https://lichess.org/

**Comment by StuSerious** [comment]
I looked a bit into theming for lichess and it seems it's not as straightforward as I initially imagined, something like Stylus would be most definitely required.

The advantage is that there are lots of themes already out there (on userstyles for example) so there is at least material to use as template.

Another factor to consider would be the board/piece theming and how to handle that correctly.. 
Which colors of the palettes go on what elements of the UI?
Do we want to theme pieces according to user accents?

I'm working on a sketch of how it would look, but I'd definitely appreciate some feedback and direction here.

@MAHcodes I think a post in the discord would be a good starting point to discuss this, what do you think?

**Comment by MAHcodes** [comment]
Hey @StuSerious, appreciate your comment, yes I think Stylus is the way to go, I'd prefer to do so, we need to check other styles as you said, we def have to make some decisions and keep the rest for the user preference if possible.
And sure, feel free to post in the discord when you have the sketch ready, and I'll be playing around with some styles, so we can easily apply your sketch for the board/piece.

**Comment by NamelessGO** [comment]
Would be possible to create a catppuccin profile for https://prettierlichess.github.io/ ?

Places to add Catppuccin theme
- https://github.com/algertc/prettierlichess-themes
- https://drive.google.com/drive/folders/1QJ0d5MAtUkYnp81l_VBJQwDcDm_oZPav
<img width="340" alt="image" src="https://github.com/catppuccin/catppuccin/assets/66227691/f461c827-38b9-49c9-9d4c-2cc0edbcf0a7">
<img width="345" alt="image" src="https://github.com/catppuccin/catppuccin/assets/66227691/504c1737-90fd-4498-a5fa-8a844b1e80e7">

**Comment by sgoudham** [comment]
Closing as it now exists: https://github.com/catppuccin/userstyles/tree/main/styles/lichess
